test/services/videoconference/logic/index.test.js

```diff
@@ -0,0 +1,81 @@
+const { expect } = require('chai');
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)